libs/orchestrator-ui-components/src/lib/components/Table/columns.ts

```diff
@@ -0,0 +1,38 @@
+import { EuiBasicTableColumn } from '@elastic/eui';
```

Comment:
I moved this file to libs/orchestrator-ui-components/src/lib/components/Table/utils/columns.ts